sdk/cosmos/azure-cosmos/src/main/java/com/azure/cosmos/FeedResponse.java

```diff
@@ -15,7 +18,7 @@
 import java.util.concurrent.ConcurrentHashMap;
 import java.util.concurrent.ConcurrentMap;
 
-public class FeedResponse<T> {
+public class FeedResponse<T> implements PagedResponse<T> {
```

Comment:
Is this implementing `PagedResponse` to maintain current functionality? It appears to be missing some functionality that is expected when implementing `PagedResponse`.